src/components/taskDetails/task-details.module.scss

```diff
@@ -70,6 +70,7 @@ $off-white: #f0f0f0;
 .sectionHeading {
     font-weight: bold;
     font-size: 1.2rem;
+    margin: 18px 0;
```

Comment:
Suggestion: Try to use relative units instead of absolute units.

Reply:
Made the changes